src/resources/views/partials/location.blade.php

```diff
@@ -2,19 +2,12 @@
   {{ trans('web::seat.unknown') }}
 @else
   @if(! is_null($location->solar_system))
-  <a href="//evemaps.dotlan.net/system/{{ $location->solar_system->name }}" target="_blank">
-    <span class="fas fa-map-marker-alt"></span>
-  </a>
-  @endif
-  @if(!is_null($location->structure))
+    @include("web::partials.system",["system"=>$location->solar_system->name,"security"=>$location->solar_system->security])
```

Comment:
```suggestion
    @include('web::partials.system', ['system' => $location->solar_system->name, 'security' => $location->solar_system->security])
```

use simple quote instead double ones, double quote is limisted to html usage

Reply:
I know seat uses single quotes, but I'm just used to using double quotes